edx_event_bus_kafka/internal/producer.py

```diff
@@ -186,24 +186,24 @@ def inner_to_dict(event_data, ctx=None):  # pylint: disable=unused-argument
     return key_serializer, value_serializer
 
 
-def get_headers_from_signal_and_metadata(signal: OpenEdxPublicSignal, event_metadata: EventsMetadata):
+def get_headers_from_metadata(event_metadata: EventsMetadata):
```

Comment:
Is this private? Can we add back signal when and if we need it? I really like using underscores, like `_get_headers_from_metadata`, but maybe we aren't doing that and it would be a bigger change. It just makes it so much nicer for others (like me) to come into code and no what is simpler to refactor, and what is changing an API.

Reply:
[fixed]

Reply:
jk, making this private makes it hard to test, and since it's outside the producer class proper hopefully it's clear that it's not part of the API. Thoughts?